src/components/content/Content.js

```diff
@@ -1,24 +1,36 @@
-import List from '../list/index'
+import Greetings from '../greetings/index';
+import Numbers from '../numbers/index';
+import UsersTable from '../userstable/index';
+
 import './content.scss';
-const Error = (props) => {
-  console.log(props);
-  const date = props.date ? props.date.toLocaleString() : '';
-  const style = {backgroundColor: 'red', color: props.color || 'white'};
 
-  return <mark style={style}>There is an error: {props.text} {date}</mark>
-};
-const items = [
-  {id:1, name: 'Ilia'},
-  {id:2, name: 'Petro'}
+const list = [
+  {
+    firstName: 'Erik',
+    lastName: 'Cartman',
+    age: 12
+  },
+  {
+    firstName: 'Kyle',
+    lastName: 'Broflovsky',
+    age: 12
+  },
+  {
+    firstName: 'Stan',
+    lastName: 'Marsch',
+    age: 12
+  },
 ];
 
 const Content = () => (
   <div className="content">
     <h2> Content </h2>
-    <p>Lorem ipsum</p>
-    <List items={items}/>
-    <List/>
+    <Greetings time={new Date().getHours()} name="Alex" />
```

Comment:
component `Greetings` should encapsulate time logic inside its implementation (don't pass time via attributes)